app/models/mixins/authentication_mixin.rb

```diff
@@ -12,6 +12,33 @@ def self.authentication_check_schedule
       assocs = zone.respond_to?(assoc) ? zone.send(assoc) : []
       assocs.each { |a| a.authentication_check_types_queue(:attempt => 1) }
     end
+
+    def self.queue_authentication_check(args, user_id, zone)
```

Comment:
@jntullo I think those numbers are not bad, but in a real-world situation there is good chance that it could take much longer (could even be several minutes, I think), which the users will surely notice, given the fact that they are used to an instantaneous response after clicking the Validate button.
Hence I wanted to suggest keeping this queued validation restricted to only the use case specified in the BZ. 
(We are already discussing this approach in the UI PR: https://github.com/ManageIQ/manageiq-ui-classic/pull/1580#issuecomment-320115947)